### PR TITLE
feat(*): support for parsing manifest files

### PR DESCRIPTION
**Pull request by technosophos**
This provides the basic utilities for working with Kubernetes manifests.

### CODE REVIEW

File: helm/manifest/manifest.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/registered"
+	_ "k8s.io/kubernetes/pkg/api/v1"
+	_ "k8s.io/kubernetes/pkg/apis/experimental"
+	"k8s.io/kubernetes/pkg/util/yaml"
+)
+
+// Files gets a list of all manifest files inside of a chart.
+//
+// chartDir should the the path to a chart (the directory which
```
Comment (mboersma) [inline helm/manifest/manifest.go]: typo: I assume this should be `chartDir will return the path to a chart`

File: helm/manifest/manifest.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	return files, nil
+}
+
+// Manifest represents a Kubernetes manifest object.
+type Manifest struct {
+	Version, Kind   string
+	VersionedObject interface{}
+}
+
+func Parse(filename string) (*Manifest, error) {
```
Comment (mboersma) [inline helm/manifest/manifest.go]: Should have a doc comment here since it's an exported func.

**Comment by technosophos** [comment]
Fixed both of the issues @mboersma noted. Also removed some commented code and some needless debugging.

**Comment by gabrtv** [comment]
LGTM. Yay for reusability.

**Pull request merged**